addons/l10n_hu/i18n/hu.po

```diff
@@ -16,45 +16,36 @@ msgstr ""
 "Plural-Forms: \n"
 
 #. module: l10n_hu
-#: model:account.tax,description:l10n_hu.1_FKKT
-#: model:account.tax,description:l10n_hu.1_VKKT
-#: model:account.tax,description:l10n_hu.2_FKKT
-#: model:account.tax,description:l10n_hu.2_VKKT
-#: model:account.tax,name:l10n_hu.1_FKKT model:account.tax,name:l10n_hu.1_VKKT
-#: model:account.tax,name:l10n_hu.2_FKKT model:account.tax,name:l10n_hu.2_VKKT
-#: model:account.tax.template,description:l10n_hu.FKKT
-#: model:account.tax.template,description:l10n_hu.VKKT
-#: model:account.tax.template,name:l10n_hu.FKKT
-#: model:account.tax.template,name:l10n_hu.VKKT
-msgid "0% Goods Exempt"
-msgstr "0% árumentes"
-
-#. module: l10n_hu
-#: model:account.tax,description:l10n_hu.1_FEXT
-#: model:account.tax,description:l10n_hu.2_FEXT
-#: model:account.tax,name:l10n_hu.1_FEXT model:account.tax,name:l10n_hu.2_FEXT
-#: model:account.tax.template,description:l10n_hu.FEXT
-#: model:account.tax.template,name:l10n_hu.FEXT
-msgid "0% Goods Export"
-msgstr "0% áruexport"
-
-#. module: l10n_hu
-#: model:account.tax,name:l10n_hu.1_FEUT model:account.tax,name:l10n_hu.2_FEUT
-#: model:account.tax.template,name:l10n_hu.FEUT
-msgid "0% Goods ICD"
-msgstr "0% áruk ICD"
+#: model:account.tax,description:l10n_hu.1_FA
+#: model:account.tax,description:l10n_hu.1_VA
+#: model:account.tax,description:l10n_hu.2_FA
+#: model:account.tax,description:l10n_hu.2_VA
+#: model:account.tax,name:l10n_hu.1_FA model:account.tax,name:l10n_hu.1_VA
+#: model:account.tax,name:l10n_hu.2_FA model:account.tax,name:l10n_hu.2_VA
+#: model:account.tax.template,description:l10n_hu.FA
+#: model:account.tax.template,description:l10n_hu.VA
+#: model:account.tax.template,name:l10n_hu.FA
+#: model:account.tax.template,name:l10n_hu.VA
+msgid "0% AAM"
+msgstr ""
```

Comment:
Didn't add a translation after reading this 
![image](https://github.com/user-attachments/assets/279d6fc4-6f93-4bdf-a0ed-75705f398b69)
But if you think it's better to add translations here, I will add it :)

Reply:
It's great like this, thanks!